src/rpdk/core/test.py

```diff
@@ -52,14 +54,46 @@ def temporary_ini_file():
         yield str(path)
 
 
-def get_overrides(root):
+def get_cloudformation_exports(region_name, endpoint_url):
+    session = create_sdk_session(region_name)
+    cfn_client = session.client("cloudformation", endpoint_url=endpoint_url)
+    paginator = cfn_client.get_paginator("list_exports")
+    pages = paginator.paginate()
+    exports = {}
+    for page in pages:
+        exports.update({export["Name"]: export["Value"] for export in page["Exports"]})
+    return exports
+
+
+def render_jinja(overrides_string, region_name, endpoint_url):
+    env = Environment(autoescape=True)
+    parsed_content = env.parse(overrides_string)
+    variables = meta.find_undeclared_variables(parsed_content)
+    if variables:
+        exports = get_cloudformation_exports(region_name, endpoint_url)
+        invalid_exports = variables - exports.keys()
+        if len(invalid_exports) > 0:
+            invalid_exports_message = (
+                "Override file invalid: %s are not valid cloudformation exports."
+                + "No Overrides will be applied"
+            )
+            LOG.warning(invalid_exports_message, invalid_exports)
+            return empty_override()
```

Comment:
why don't we fail immediately when this happens?

Reply:
I was keeping with the current behavior of cfn test. If there are any issues with the overrides template in general, we will simply not apply any overrides and try to execute the tests. If we want to change it, it probably shouldn't be part of this pr.

Reply:
seems weird to me but okay